input/includes/prov-white-list.md

```diff
@@ -17,5 +15,4 @@ AU Core Provenance **SHALL** be supported for the following resource types that
 <li>PractitionerRole</li>
 <li>Procedure</li>
 <li>RelatedPerson</li>
```

Comment:
RelatedPerson shouldn't be here.

Reply:
RelatedPerson is removed now from the list